viite-UI/src/view/ProjectEditForm.js

```diff
@@ -251,6 +251,8 @@
         selectedProjectLink = false;
         selectedProjectLinkProperty.cleanIds();
         rootElement.html(emptyTemplate(projectCollection.getCurrentProject().project));
+        $('[id^=editProject]').css('visibility', 'visible');
+        $('#closeProjectSpan').css('visibility', 'visible');
```

Comment:
Use the method "formCommon.toggleAdditionalControls();" instead of change the visibility manually